File: fabric/src/main/java/juuxel/adorn/compat/BiomeMakeoverCompat.java
package juuxel.adorn.compat;

import juuxel.adorn.block.variant.BlockVariant;
import juuxel.adorn.block.variant.PrefixedBlockVariantSet;

import java.util.List;

public final class BiomeMakeoverCompat extends PrefixedBlockVariantSet {
    @Override
    public String getModId() {
        return "biomemakeover";
    }

    @Override
    public List<BlockVariant> getWoodVariants() {
        return createVariants(
            BlockVariant.Wood::new,
            "ancient_oak",
            "blighted_balsa",
            "willow",
            "swamp_cypress"
        );
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/compat/BiomeMakeoverCompat.kt
package juuxel.adorn.compat

import juuxel.adorn.api.block.BlockVariant

object BiomeMakeoverCompat : BlockVariantSet {
    override val woodVariants = listOf(
        "ancient_oak",
        "blighted_balsa",
        "willow",
        "swamp_cypress",
    ).map { BlockVariant.Wood("biomemakeover/$it") }
}
